src/Tests.Microsoft.Security.Utilities.Core/AdoLegacyPatTests.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics.CodeAnalysis;
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Security.Utilities.Core
+{
+    [TestClass]
+    public class AdoLegacyPatTests
+    {
+        [TestMethod]
+        public void AdoLegacyPat_InvalidBase32Input()
```

Comment:
>AdoLegacyPat_InvalidBase32Input [](http://example.com/codeflow?start=20&length=31)

Boo. For all my talk of test-driven development in the last PR, I ended up not shipping a correct fix! The reason? Late-breaking changes to a test I authored altered the code coverage and then at the very end a conversion from generated exception type (from `FormatException` to `ArgumentException`) left the code in a crashing condition,

Sigh. Even while waiting on a good unit test pattern, I should have started with simple, atomic tests to reveal bugs and prove they are fixed. Duly chastened, I return here with this review.